### PR TITLE
Pass paint_session as reference

### PR DESCRIPTION
**Pull request by Gymnasiast**
I got carried away, it seems. ;-)

### CODE REVIEW

File: src/openrct2/cmdline/BenchSpriteSort.cpp [hunk 1]
```diff
@@ -150,7 +150,7 @@ static void BM_paint_session_arrange(benchmark::State& state, const std::vector<
         state.PauseTiming();
         std::copy_n(local_s, std::size(sessions), sessions.begin());
         state.ResumeTiming();
-        PaintSessionArrange(&sessions[0].Session);
+        PaintSessionArrange(*(&sessions[0].Session));
```
Comment (Gymnasiast) [inline src/openrct2/cmdline/BenchSpriteSort.cpp:153]: @ZehMatt Is this correct, or should it be 
```suggestion
        PaintSessionArrange(sessions[0].Session);
```
?

Comment (ZehMatt) [reply src/openrct2/cmdline/BenchSpriteSort.cpp:153]: You are correct about that.

**Review by ZehMatt — APPROVED** [review]
Looks good to me except that one case, I don't expect any breakage with a change like that.

File: src/openrct2/cmdline/BenchSpriteSort.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2/cmdline/BenchSpriteSort.cpp:153]: Which one is correct? The code in my suggestion?

Comment (ZehMatt) [reply src/openrct2/cmdline/BenchSpriteSort.cpp:153]: Yes, the first one is doing Dereference(PointerOf(Sessions[0].Session)) which makes no sense really.

**Pull request merged**